### PR TITLE
Write test for navbar

### PR DESCRIPTION
**Pull request by anishpawaskar**
**Closes:** https://github.com/Real-Dev-Squad/members-site/issues/104

### Description
Write test for navbar desktop and mobile component

Backend changes:
- [ ] Yes
- [x] No

Frontend Changes:
- [x] Yes
- [ ] No

Database changes:
- [ ] Yes
- [x] No

Breaking changes (If your feature is breaking/missing something please mention pending tickets)
- [ ] Yes
- [x] No

**Test Coverage:**
NavbarDesktop test tsx - members-site - Visual Studio Code 23-11-2023 00_02_43

### CODE REVIEW

File: src/test/unit/components/navbar/NavbarDesktop.test.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    expect(homeLink).toBeInTheDocument();
+    expect(eventLink).toBeInTheDocument();
+    expect(memberLink).toBeInTheDocument();
+    // TODO: Uncomment when crypto is added
+    // expect(cryptoLink).toBeInTheDocument();
+    expect(statusLink).toBeInTheDocument();
+  })
+
+  test("should navigate to different route when different navbar link is clicked", async () => {
```
Comment (yesyash) [inline src/test/unit/components/navbar/NavbarDesktop.test.tsx:43]:
```suggestion
  test("should navigate to a different route when a link is pressed", async () => {
```

File: src/test/unit/components/navbar/NavbarDesktop.test.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    // const cryptoLink = screen.getByRole("link", { name: "Crypto" });
+    const statusLink = screen.getByRole("link", { name: "Status" });
+
+    expect(homeLink).toBeInTheDocument();
+    expect(eventLink).toBeInTheDocument();
+    expect(memberLink).toBeInTheDocument();
+    // TODO: Uncomment when crypto is added
+    // expect(cryptoLink).toBeInTheDocument();
+    expect(statusLink).toBeInTheDocument();
+  })
```
Comment (yesyash) [inline src/test/unit/components/navbar/NavbarDesktop.test.tsx:41]: - If I add a new link in the future should this test pass or fail?

File: src/test/unit/components/navbar/NavbarMobile.test.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+    expect(homeLink).toBeInTheDocument();
+    expect(eventLink).toBeInTheDocument();
+    expect(memberLink).toBeInTheDocument();
+    // TODO: Uncomment when crypto is added
+    // expect(cryptoLink).toBeInTheDocument();
+    expect(statusLink).toBeInTheDocument();
+  })
+
+  test("should render navbar links when user click on hamburger button and navigate to different route when different navbar link is clicked ", async () => {
```
Comment (yesyash) [inline src/test/unit/components/navbar/NavbarMobile.test.tsx:56]:
```suggestion
  test("should navigate to a different route when a link is pressed", async () => {
```

**Review by yesyash — CHANGES_REQUESTED** [review]

File: src/test/unit/components/navbar/NavbarDesktop.test.tsx (hunk 2 above)
Comment (anishpawaskar) [reply src/test/unit/components/navbar/NavbarDesktop.test.tsx]: It should fail

File: src/test/unit/components/navbar/NavbarMobile.test.tsx (hunk 3 above)
Comment (anishpawaskar) [reply src/test/unit/components/navbar/NavbarMobile.test.tsx]: Okay

**Review by yesyash — APPROVED** [review]

**Review by shubhamsigdar1 — APPROVED** [review]

**Pull request merged**